x-pack/platform/plugins/shared/streams_app/public/components/streams_app_page_template/index.tsx

```diff
@@ -27,15 +27,19 @@ StreamsAppPageTemplate.Body = ({
   <EuiPageTemplate.Section
     grow
     css={css`
+      max-height: calc(
+        100vh - var(--euiFixedHeadersOffset, 0) - 164px
```

Comment:
Unfortunately I don't think this works because the header doesn't have a fixed height:
<img width="808" height="245" alt="Screenshot 2025-10-01 at 17 47 24" src="https://github.com/user-attachments/assets/351f2b4e-a36a-4d9f-80b3-3196f2d60861" />

I'll look into it